src/commands/syncModel/run/README.md

```diff
@@ -1,101 +1,249 @@
-- Asset folders cannot be moved so if they are in a different location in the source environment, they are removed and created in the new place.
-- Asset folders cannot be deleted (or moved, see the previous point) if they contain assets. The command is not able to check this without loading all the assets in the project so it doesn't check it. Please, make sure that all folders that will be deleted does not contain any assets. You can see what folders will be deleted in the generated diff `sync-model diff ...`.
-- Languages cannot be deleted, instead, they are deactivated. Their name and codename are replaced with the first 8 characters of a randomly generated UUID (name and codename have a limit of 25 characters).
-- Roles cannot be added or patched via MAPI and are therefore not synced.
-  - Consequently, **all role restrictions for workflow steps are lost** when adding a new workflow or adjusting an existing one during sync.
+# Sync-Model Run Command
+
+> **⚠️ Caution**
+>
+> Synchronizing the content model can lead to irreversible changes in the target environment, such as:
+>
+> - **Deletion of content** by removing elements from content types.
+> - **Deletion of used taxonomies**.
+> - **Removal of role limitations** in workflows (see [Known Limitations](#known-limitations)).
+>
+> Proceed with caution and consider backing up your environment or testing in a non-production environment first.
+
+The `sync-model run` command synchronizes the **source content model** into the **target environment** using the [Kontent.ai Management API](https://kontent.ai/learn/docs/apis/openapi/management-api-v2/). This command is essential for maintaining consistency across environments (e.g., Development, Staging, Production) by updating the content model in the target environment to match the source.
+
+## Why Use `sync-model run`?
+
+- **Environment Consistency**: Ensure that content models are identical across environments, preventing issues caused by discrepancies.
+- **Deployment Automation**: Automate content model updates as part of your CI/CD pipeline.
+- **Team Collaboration**: Allow multiple team members to work on content models in different environments and synchronize changes efficiently.
+
+## Source Content Model Options
+
+You can obtain the source content model in two ways:
+
+1. **From an Existing Kontent.ai Environment**: Provide the source environment ID and Management API key to fetch the content model directly.
+2. **From a Local Folder**: Use a local folder containing the content model exported using the [`sync-model export`](../export/README.md) command.
+
+**Using a local snapshot of the content model is the recommended approach**, as it creates a stable snapshot that won't change during the synchronization process. This prevents discrepancies that can occur if changes are made in the source environment between diffing and syncing.
```

Comment:
between diffing with a command.. diff that is printed during sync-model run is ok

Reply:
tru!